hedera-node/src/main/java/com/hedera/services/fees/calculation/utils/OpUsageCtxHelper.java

```diff
@@ -85,6 +88,33 @@ public ExtantFeeScheduleContext ctxForFeeScheduleUpdate(TokenFeeScheduleUpdateTr
 		return new ExtantFeeScheduleContext(token.expiry(), curFeeScheduleReprSize(token.customFeeSchedule()));
 	}
 
+	public ExtantCryptoContext ctxForCryptoUpdate(TransactionBody txn) {
+		final var op = txn.getCryptoUpdateAccount();
+		ExtantCryptoContext cryptoContext;
+		var info = workingView.infoForAccount(op.getAccountIDToUpdate());
+		if (info.isPresent()) {
+			var details = info.get();
+			cryptoContext = ExtantCryptoContext.newBuilder()
+					.setCurrentKey(details.getKey())
+					.setCurrentMemo(details.getMemo())
+					.setCurrentExpiry(details.getExpirationTime().getSeconds())
+					.setCurrentlyHasProxy(details.hasProxyAccountID())
+					.setCurrentNumTokenRels(details.getTokenRelationshipsCount())
+					.setCurrentMaxAutomaticAssociations(details.getMaxAutomaticTokenAssociations())
+					.build();
+		} else {
+			cryptoContext = ExtantCryptoContext.newBuilder()
```

Comment:
If the account is not found in MerkleMap, does this mean that the account doesn't exist (deleted or wrong account id)? or it's still in the middle of creation?. If it's the first case, seems like it doesn't make sense to continue with the update transaction.